### DISCUSSION TITLE
[求助]纯电队怎么练？

### DISCUSSION THREAD

**Discussion by kisakiIMO** [share121/inter-knot#2916] [其他]
抽青衣出了个丽娜，正好跟我1+0格里斯组纯电队，目前丽娜4自由蓝调+2穿透。看别人伤害很高，我却打得比自己之前的紊乱队还慢，按理说有了丽娜数值就不成问题，这是怎么回事捏<img src="https://img-static.mihoyo.com/communityweb/upload/e1d38f0a36654df2920a692f26f0670a.png" alt="甘雨-懊恼" width="75" >
<img width="524" alt="屏幕截图 2024-08-19 005552" src="https://github.com/user-attachments/assets/d0bfa905-152b-4593-91fb-a9e89c101b54">
<img width="530" alt="屏幕截图 2024-08-19 005523" src="https://github.com/user-attachments/assets/100f6d4c-8e20-4c31-846c-37df2f385f8f">

**Comment by Onefeilinmiao** [comment]
1命丽娜最好，不过0命也能用，另一个队友带露西，格莉丝带4电2穿透/2蓝调，5号位换穿透，6号位如果你能全程感电就换攻击，不能就带掌握

**Comment by kisakiIMO** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/54f6c5f6a4bf851877158b0000e636c5.png" alt="克拉拉 给你心心" width="75" >
